tests/unit/readers/cubeconfig/v1_0/test_cube_config_extra_cols.py

```diff
@@ -23,7 +23,7 @@ def test_build_with_fail():
                 output_directory=output,
                 csv_path=csv,
                 fail_when_validation_error_occurs=True,
-                validation_errors_file_out=Path(output, "validation_errors.json"),
+                validation_errors_file_name=Path(output, "validation_errors.json"),
```

Comment:
Again, shouldn't this just be the file name now? There are other lines like this in the same file.